doc(Pages): Update remaining pages to remove reference to Portal (#1737)
bonitasoft/bonita-doc: modules/ROOT/pages/run-a-process-from-bonita-bpm-studio-for-testing.adoc

* Open the process diagram and select the pool, then click *_Run_* in the Cool bar. The process in the selected pool runs. Only one pool can be run at a time.

If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can select the environment in which to run the process. The

> process must be configured for the environment in which it runs.

Reviewer: If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can select the environment in which to run the process. The process must be configured for the environment in which it runs. =>  A process can only run in the environment(s) in which it has been configured. If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can choose the environment in which to run the process.

> If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can select the environment in which to run the process. The
> process must be configured for the environment in which it runs.

Reviewer 2: ```suggestion
A process can only run in the environment(s) in which it has been configured. If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can choose the environment in which to run the process.
```
